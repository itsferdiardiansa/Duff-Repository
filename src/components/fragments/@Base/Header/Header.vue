<template>
  <section class="app-header">
    <div class="app-header--container">
      <Navbar />
    </div>
  </section>
</template>
<script>
// import SearchForm from './SearchForm'
import Navbar from './Navbar'

export default {
  components: {
    // SearchForm,
    Navbar,
  },
}
</script>
<style lang="scss" scoped>
.app-header {
  @apply fixed bg-indigo-800 w-full z-40 box-border;
  top: 0;
  height: $pageHeaderHeight;

  &--container {
    @apply flex h-full px-6 items-center;
  }
}
</style>
