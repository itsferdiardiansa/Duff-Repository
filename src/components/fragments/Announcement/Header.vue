<template>
  <div class="content-header flex items-center justify-between">
    <div class="flex items-center">
      <div class="title">
        <h2 class="font-bold text-2xl">Announcement</h2>
      </div>
    </div>

    <div class="btn-create float-right">
      <Button :textBold="true" @click="createAnnouncement" variant="primary">
        Create
      </Button>
    </div>
  </div>
</template>
<script>
import { useRouter } from 'vue-router'
import Button from '@common/Button'

export default {
  components: {
    Button,
  },
  setup() {
    const router = useRouter()

    const createAnnouncement = () => {
      router.push('/announcement/create')
    }

    return {
      createAnnouncement,
    }
  },
}
</script>
