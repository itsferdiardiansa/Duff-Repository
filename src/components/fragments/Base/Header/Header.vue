<template>
  <section class="page-header">
    <div class="flex h-full px-6 items-center">
      <SearchForm />

      <Navbar />
    </div>
  </section>
</template>
<script>
import SearchForm from './SearchForm'
import Navbar from './Navbar'

export default {
  components: {
    SearchForm,
    Navbar
  }
}
</script>
<style lang="scss" scoped>
.page-header {
  @apply h-16 fixed bg-indigo-800 w-full z-40 box-border;
  top: 0;
  padding-left: 255px;
}
</style>