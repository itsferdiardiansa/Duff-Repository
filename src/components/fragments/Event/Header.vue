<template>
  <div class="content-header flex items-center justify-between">
    <div class="flex items-center">
      <div class="title">
        <h2 class="font-bold text-2xl">List Event</h2>
      </div>
    </div>

    <div class="float-right flex">
      <TextField class="mr-4 border border-gray-400" />

      <Button :textBold="true" class="w-40" variant="primary">
        Search Event
      </Button>
    </div>
  </div>
</template>
<script>
import Button from '@common/Button'
import TextField from '@common/TextField'

export default {
  components: {
    Button,
    TextField,
  },
}
</script>
