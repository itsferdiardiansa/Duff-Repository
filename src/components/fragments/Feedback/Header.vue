<template>
  <div class="content-header flex items-center justify-between">
    <div class="flex items-center">
      <div class="title">
        <h2 class="font-bold text-2xl">Feedback Data</h2>
      </div>
    </div>
  </div>
</template>
<script>
export default {
  components: {},
}
</script>
