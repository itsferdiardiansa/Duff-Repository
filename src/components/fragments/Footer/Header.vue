<template>
  <div class="content-header flex items-center justify-between">
    <div class="flex items-center">
      <div class="title">
        <h2 class="font-bold text-2xl">Footer Data</h2>
      </div>
    </div>

    <div class="btn-create float-right">
      <Button :textBold="true" @click="createFooter" variant="orange">
        Create
      </Button>
    </div>
  </div>
</template>
<script>
import { useRouter } from 'vue-router';
import Button from '@common/Button';

export default {
  components: {
    Button,
  },
  setup() {
    const router = useRouter();

    const createFooter = () => {
      router.push('/footer/create');
    };

    return {
      createFooter,
    };
  },
};
</script>
