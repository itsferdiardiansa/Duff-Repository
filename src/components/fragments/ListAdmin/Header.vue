<template>
  <div class="content-header flex items-center justify-between">
    <div class="flex items-center">
      <div class="title">
        <h2 class="font-bold text-2xl">List Admin CMSLOKET</h2>
      </div>
    </div>

    <div class="btn-create float-right">
      <Button 
        variant="primary"
        :icon="['fa', 'plus']"
        :bold="true" 
        @click="addAdmin" 
      >
        Add Admin
      </Button>
    </div>
  </div>
</template>
<script>
import { useRouter } from 'vue-router'
import Button from '@common/Button'

export default {
  components: {
    Button,
  },
  setup() {
    const router = useRouter()

    const addAdmin = () => {
      router.push('/list-admin/add')
    }

    return {
      addAdmin,
    }
  },
}
</script>
