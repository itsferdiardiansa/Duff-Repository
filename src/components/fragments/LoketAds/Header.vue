<template>
  <div class="content-header flex items-center justify-between">
    <div class="flex items-center">
      <div class="title">
        <h2 class="font-bold text-2xl">Loket Ads</h2>
      </div>
    </div>

    <div class="btn-create float-right">
      <Button :textBold="true" @click="createLoketAds" variant="primary">
        Create
      </Button>
    </div>
  </div>
</template>
<script>
import Button from '@common/Button'
import { useRouter } from 'vue-router'

export default {
  components: {
    Button,
  },
  setup() {
    const router = useRouter()

    const createLoketAds = () => {
      router.push('/loket-ads/create')
    }

    return {
      createLoketAds,
    }
  },
}
</script>
