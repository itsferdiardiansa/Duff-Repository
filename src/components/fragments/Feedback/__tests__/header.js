import { mount } from '@vue/test-utils';
import Header from '@fragment/Feedback/Header';

describe('fragment/Feedback/Header', () => {
  it('will render and match snapshot', async () => {
    const Component = mount(<Header />);

    expect(Component.find('h2').html()).toContain('Feedback Data');
    expect(Component.html()).toMatchSnapshot();
  });
});
