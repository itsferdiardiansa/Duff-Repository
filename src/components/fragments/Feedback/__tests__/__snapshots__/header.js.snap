// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fragment/Feedback/Header will render and match snapshot 1`] = `
<div class="content-header flex items-center justify-between">
  <div class="flex items-center">
    <div class="title">
      <h2 class="font-bold text-2xl">Feedback Data</h2>
    </div>
  </div>
</div>
`;
